docs/crafted-startup.org

```diff
@@ -0,0 +1,45 @@
+* Crafted Emacs Startup Module
+
+** Installation
+
+To use this module, simply require them in your =init.el= at the appropriate
+points.
+
+#+begin_src emacs-lisp
+;; No additional packages are installed by this module.
+
+;; install the packages
+(package-install-selected-packages :noconfirm)
+
+;; Load crafted-startup configuration
+(require 'crafted-startup-config)
+#+end_src
+
+** Description
+
+The =crafted-startup= module provides a fancy splash screen similar to the Emacs
+default splash screen or the Emacs about page.
+
+If it is used together with the =crafted-updates= module, it will also check for
```

Comment:
This may need an example source block similar to elucidate the issue.

Reply:
Good call. Done.